tempesta_fw/http_sticky.c

```diff
@@ -316,9 +307,11 @@ tfw_http_sticky_found(TfwHttpMsg *hm, TfwStr *value)
 	/*
 	 * Do nothing for now. The request is passed to a backend server.
 	 */
-	/* XXX This assumes that 'value' is a linear TfwStr{}. */
-	TFW_DBG("Sticky cookie found: \"%.*s\"\n",
-		(int)value->len, (char *)value->ptr);
+	if (likely(TFW_STR_PLAIN(value)))
```

Comment:
The downside of this is that when debugging is compiled out that will leave an empty `if()` block. Perhaps it's better to print full or partial value depending on whether the string is compound or not, and do all of that within `TFW_DBG()` statement.